src/Client.js

```diff
@@ -327,7 +350,24 @@ Client.prototype._execute = function(method, path, data, query, options) {
       )
       self._handleRequestResult(response, result, options)
 
-      return responseObject['resource']
+      const metricsHeaders = [
+        'x-compute-ops',
+        'x-byte-read-ops',
+        'x-byte-write-ops',
+        'x-query-time',
+        'x-txn-retries',
+      ]
+
+      if (options && options.metrics) {
+        return {
+          value: responseObject['resource'],
+          metrics: Object.fromEntries(Array.from(Object.entries(response.headers)).
+            map(([ k,v ]) => [k, parseInt(v)]).
+            filter( ([k,v]) => metricsHeaders.includes(k) )),
```

Comment:
nit - filter and then map, save some computation
```suggestion
            filter( ([k,v]) => metricsHeaders.includes(k) )).
            map(([ k,v ]) => [k, parseInt(v)])
            ,
```